### ISSUE TITLE
Primary Node Reboot - Mimic Linksys Velop App Restart Mesh Network

### ISSUE THREAD

**Issue by Karun12234** [uvjim/linksys_velop#455]
Hello:

I was wondering if you can put a button like there is for all the secondary node to reboot the main node instead of calling a service etc.  As it doesn’t seem to work.

The reason I’m asking this is because when I try to reboot with is primary and turn the flag on, it doesn’t seem to reboot like if I press the button on the secondary nodes those reboot.

Basically, if there could be a button to mimic what you do in the Linksys velop  app, and you click restart mesh network, if you could have that same feature within the repository as the button, that would be really helpful.

This would save me from buying a programmable timer

Thank you!

![IMG_9304](https://github.com/uvjim/linksys_velop/assets/166978143/acf793aa-1294-4288-9f4b-9c2867b0bce9)
![IMG_9303](https://github.com/uvjim/linksys_velop/assets/166978143/c94a5a49-7e69-4c1b-bdae-2230acc3cb19)

**Comment by uvjim** [comment]
What do you mean when you say starting the primary node doesn't seem to work?

As far as I can tell it works. I recently released a new version that fixed an issue with the buttons and also fixed an issue where the wrong IP address was used for addressing the secondary nodes.

The service and the buttons essentially follow the same code path.

**Comment by Karun12234** [comment]
Basically, on the secondary nodes, when you click the button in Home
Assistant, to click reboot, you see the light on the note, blink, telling
me that it’s resetting it’ll go from blue to red and blue. On the primary
note when I call a service to reboot it it doesn’t blink or anything
doesn’t happen. Then when I checked the logs, for the primary note, there’s
nothing there for a reboot. But if I do the same reboot on a secondary
note, and I checked the log to see if there was a reboot done so basically
what I’m trying to do is reboot, every single note in my house at the same
time.  And this can be done there is something that mimics restart the mesh
network or restart the network within the Linksys app

**Comment by Karun12234** [comment]
See I started a reboot by calling a service for the secondary node 

You will see it blinking blue red the primary doesn’t do this .


![IMG_9307](https://github.com/uvjim/linksys_velop/assets/166978143/b3007cc2-52ea-42fa-a15a-d80ebd39cb3b)
![IMG_9308](https://github.com/uvjim/linksys_velop/assets/166978143/8e1d838d-c137-4c9c-a296-f2757d4d9fc4)
![IMG_9311](https://github.com/uvjim/linksys_velop/assets/166978143/46bea624-e8b0-4219-b1c8-5f59457a4df1)

**Comment by uvjim** [comment]
Could you enable debug logging please?  As far as I know, when I last tested it, it worked fine.  I'm not in a position to force the mesh to reboot at the moment to check it - I must find the nodes I was using for testing...

Also, I'm not really sure about adding a button to do the restart to the primary node or the mesh service.  The problem being that buttons in HASS don't have a confirm option (they do on the dashboard but not in the Devices area).  That means you could easily, and accidentally) reboot the mesh when you didn't want to.

**Comment by Karun12234** [comment]
It’s fine.

I ended up getting a programable timer.

Thank you for your help.

**Comment by PaRkThEcAr** [comment]
Hi! i am getting this error on my MBE7000's
<img width="483" alt="image" src="https://github.com/uvjim/linksys_velop/assets/39027440/f4f2f428-a0ae-4bea-b6c3-d9bda9e608cf">
this is the error from the log. it says there is no address for the node. however, i reconfigured the integration and am still having the issues

**Comment by uvjim** [comment]
@PaRkThEcAr - can you raise this as a separate issue please.  It doesn't relate to this one at all.

**Comment by uvjim** [comment]
@Karun12234  - I'm not sure what is happening for you, but I just traced the call used by the app for the that button and it is exactly the same as the JNAP request that is used by the upstream library.

**Comment by Karun12234** [comment]
I think what it is it has to do with link says itself. There was too many
failed password calls, and now they’ve blocked it so even though I reset it
for some reason it’s blocked on Linksys, but I can access the node through
the app. It’s weird so once I contact Linksys for about this, I’m sure I’ll
work so it’s on Linksys end
